tools/train_model.py

```diff
@@ -258,13 +292,13 @@ def check_train_val(key, name):
     parser.add_argument(
         "--cfg",
         dest="cfg_file",
-        help="Config file for training.",
+        help="Config file for training. Can be either a local file or a remote URL.",
         required=True,
-        type=pathlib.Path,
+        type=file_or_url,
     )
     parser.add_argument(
         "--initialization-checkpoint",
-        type=pathlib.Path,  # noqa
+        type=file_or_url,
         help="If this value is set to a proper checkpoint when training starts, "
         "the model will be initialized with the weights given. "
         "No other keys in the checkpoint will be loaded. "
```

Comment:
I would add a bit more elaborate description about loading from directory or downloading from a url. (Maybe in the README?)

Reply:
I have adjusted it.